style.css

```diff
@@ -613,6 +614,7 @@ textarea::placeholder {
   .see-project:hover {
     background-color: var(--primary-b-500, #396df2);
     color: white;
+    box-shadow: 0 8px 16px 0 rgba(64, 83, 252, 0.24);
   }
 
   /* About section */
```

Comment:
Please kindly reduce the width of the button in your `contact` form so that it'd look closely similar to the design 😄 

Your design :

![image](https://github.com/ayaderaghul/microverse-portfolio/assets/20537712/2ca6a94f-d126-4e20-847e-336dd60529de)

Design given :

![image](https://github.com/ayaderaghul/microverse-portfolio/assets/20537712/89747efd-dea8-4f52-89a9-b0ac4f9a91c4)